### PR TITLE
New Exercise: Phone Number

### PR DESCRIPTION
**Pull request by marianfoo**
## Source
https://github.com/exercism/javascript/tree/main/exercises/practice/phone-number

## Unit Tests in SAP

image

## ABAP Lint

`./exercises/practice/phone-number/zcl_phone_number.clas.testclass.abap[1, 7] - Place local testclass "ltcl_phone_number" in the testclass include (local_testclass_location) `

## ABAP Docker Run Test

No errors
image

### CODE REVIEW

**Comment by marianfoo** [comment]
@larshp can please help with this error.
I dont really know what it means
`./exercises/practice/phone-number/zcl_phone_number.clas.testclass.abap[1, 7] - Place local testclass "ltcl_phone_number" in the testclass include (local_testclass_location)`
Or is it something to exclude for checking?

**Comment by pokrakam** [comment]
@marianfoo the test class filename is `.testclasses.abap` not `.testclass.abap`, should work then.

**Comment by marianfoo** [comment]
@pokrakam Thanks! Works now

File: abaplint_exercises.json [hunk 1]
```diff
@@ -329,7 +329,7 @@
       "exception": "^LCX_.+$",
       "test": "^LT.+$"
     },
-    "local_testclass_location": true,
+    "local_testclass_location": false,
```
Comment (larshp) [inline abaplint_exercises.json:332]:
```suggestion
    "local_testclass_location": true,
```

File: abaplint_solutions.json [hunk 2]
```diff
@@ -325,7 +325,7 @@
       "exception": "^LCX_.+$",
       "test": "^LT.+$"
     },
-    "local_testclass_location": true,
+    "local_testclass_location": false,
```
Comment (larshp) [inline abaplint_solutions.json:328]:
```suggestion
    "local_testclass_location": true,
```

**Comment by marianfoo** [comment]
@larshp ready to merge
Where did the error come from?

**Pull request merged**